spec/factories/design_factor.rb

```diff
@@ -1,6 +1,6 @@
 FactoryGirl.define do
   factory :design_factor do
-    sequence(:design_factor_name) {|n| "#{Faker::Lorem.characters(5)}_#{n}"}
+    sequence(:design_factor_name) {|n| "#{Faker::Lorem.characters(20)}_#{n}"}
```

Comment:
You don't have to do that. If you use the sequence's "n" inside the value, factory girl should guarantee uniqueness. So 5 is enough.